src/lift3.js

```diff
@@ -0,0 +1,29 @@
+var ap = require('./ap');
+var curryN = require('./curryN');
+var map = require('./map');
+
+
+/**
+ * Promotes a curried ternary function to a function which operates on three
+ * [Apply](https://github.com/fantasyland/fantasy-land#apply)s.
+ *
+ * This function is derived from `map` and `ap`.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig Apply f => (a -> b -> c -> d) -> f a -> f b -> f c -> f d
+ * @param {Function}
+ * @param {*}
+ * @param {*}
+ * @param {*}
+ * @return {*}
```

Comment:
`@param` and `@return` really are worthless for parametrically polymorphic functions. I'd love to get rid of this cruft. :smile: